Resources/Shaders/DearImGui-Mesh.shader

```diff
@@ -22,6 +22,28 @@
         }
     }
 
+	// shader for HD render pipeline
+	SubShader
+	{
+		Tags { "RenderType" = "Transparent" "RenderPipeline" = "HDRenderPipeline" "PreviewType" = "Plane" }
```

Comment:
Just a totally uneducated guess: have you tried changing "Transparent" to "Opaque"?